### PR TITLE
Restore to working order & minor additions

### PR DESCRIPTION
**Pull request by LarsSelbekk**
The following has changed:

- Updated code to function with newer versions of Puppeteer
- Fixed zombie chrome processes by adding dumb-init
- Added explicit waiting for the input elements to load because onLoad didn't seem to be enough
- Changed NTNU logo to look correct in Slack
- Added Docker Compose configuration for even easier deployment, with .env-file
- Added option to filter on items that can be taken home
- Added optional debug-logging using DEBUG environment variable

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -74,7 +95,7 @@ let isFirst = true;
           text: txt,
           unfurl_links: 1,
           icon_url:
-            "https://innsida.ntnu.no/c/wiki/get_page_attachment?p_l_id=22780&nodeId=24647&title=Bruksregler+for+NTNU-logoen&fileName=logo_norsk_uten_slagord.jpg",
+            "http://engineering-team.net/wp-content/uploads/2015/12/logo_ntnu.png",
```
Comment (odinuge) [inline main.js]: Not too fan of hardcoding a `engineering-team.net` url here.

Can you find something else, or make it an env var, or base64 it or something?

**Review by odinuge — APPROVED** [review]
Awesome, thanks!

Good that it is in use.. 🎉 

Small nit about the logo, otherwise good to go; will merge when fixed

Thanks
Odin

**Comment by odinuge** [comment]
And big +1 for good commits with real and good titles and commit messages. I don't see that too often 👌

File: main.js (hunk 1 above)
Comment (LarsSelbekk) [reply main.js]: I totally agree, didn't think of that. I couldn't find a good one directly from NTNU, but I changed it so the link points to a capture by the Internet Archive. Is that ok?

**Pull request merged**